tests/go.mod

```diff
@@ -2,16 +2,15 @@ module github.com/icinga/icingadb/tests
 
 go 1.16
 
-replace github.com/icinga/icingadb => ../
+replace github.com/icinga/icinga-testing => github.com/icinga/icinga-testing v0.0.0-20220428124602-54e951e968d8
```

Comment:
Dang it! Missed to check this again before doing my final ready to merge check now: The PR over at icinga-testing was merged, so this should just update the version to the latest commit on the main branch in the normal require directive below.

Reply:
Well, I could have noticed that too 😆